include/rmm/detail/dynamic_load_runtime.hpp

```diff
@@ -116,6 +121,28 @@ struct async_alloc {
     return runtime_supports_pool and driver_supports_pool;
   }
 
+  /**
+   * @brief Check whether the specified `cudaMemAllocationHandleType` is supported on the present
+   * CUDA driver/runtime version.
+   *
+   * @note This query was introduced in CUDA 11.3 so on CUDA 11.2 this function will only return
+   * true for `cudaMemHandleTypeNone`.
+   *
+   * @param handle_type An IPC export handle type to check for support.
+   * @return true if supported
+   * @return false if unsupported
+   */
+  static bool is_export_handle_type_supported(cudaMemAllocationHandleType handle_type)
+  {
+    int supported_handle_types_bitmask{};
+#if CUDART_VERSION >= 11030  // 11.3 introduced cudaDevAttrMemoryPoolSupportedHandleTypes
+    cudaDeviceGetAttribute(&supported_handle_types_bitmask,
```

Comment:
Need to error check return value.

Reply:
Done, using `RMM_CUDA_TRY`. At least this way we will throw a `rmm::cuda_error` exception.